config/activityLog.js

```diff
@@ -54,6 +54,23 @@ module.exports = {
     name: 'None',
     value: 'None',
   },
+  event: {
+    activity: 'event',
+    action_id: 'None',
+    actionSuccess: 'None',
+    botName: 'None',
+    ipAddress: 'None',
+    dbTime: '0ms',
+    method: 'None', // one of HTTP verbs
+    process: 'None',
+    request_id: 'None',
```

Comment:
"request_id" is lowercase and has underscore because it is "inherited" from heroku.

If there is no good reason for "action_id" to be "action_id" then I would suggest it should be camel-case "actionId"